tests/main/desktop-portal-open-file/task.yaml

```diff
@@ -12,6 +12,7 @@ description: |
 systems:
   - ubuntu-18.04-*
   - ubuntu-19.04-*
```

Comment:
We could do "ubuntu-19*, ubuntu-2*" here and in similar situations - this will give us peace of mind for the next 10y :)